identity/app/views/profileForms.scala.html

```diff
@@ -102,9 +102,9 @@ <h1 class="identity-title" data-test-id="edit-profile-header">Edit your profile<
 
                         @content(4, "/digitalpack/edit")(profile.digitalPackDetailsForm(idUrlBuilder, idRequest, user))
 
-                        @content(5, "/contribution/recurring/edit")(profile.recurringContributionDetailsForm(idUrlBuilder, idRequest, user))
+                        @content(5, "/contribution/recurring/edit", requiresValidation)(profile.recurringContributionDetailsForm(idUrlBuilder, idRequest, user))
 
-                        @content(6, "/email-prefs")(profile.privacyForm(idUrlBuilder, idRequest, user, forms.privacyForm, emailPrefsForm, emailSubscriptions, availableLists, consentsUpdated, consentHint))
+                        @content(6, "/email-prefs", requiresValidation)(profile.privacyForm(idUrlBuilder, idRequest, user, forms.privacyForm, emailPrefsForm, emailSubscriptions, availableLists, consentsUpdated, consentHint))
```

Comment:
noticed that I wasn't correctly changing the content of the email prefs tab for non-validated users - this fixes that and adds contributions